README.MD

```diff
@@ -0,0 +1,21 @@
+[GitHub] Calculate letters/find amount of the vowel letters/deadlock
```

Comment:
This file is a copy paste from previous project. Read it and fix it

Reply:
Pull request naming is incorrect. It should describe what exactly was implemented, not just "implemented 4 tasks"
https://namingconvention.org/git/pull-request-naming.html

Reply:
Same "[Edit all comments from GitHub associated to synchronized methods](https://github.com/andreybeloboky/Multithreading/pull/1/commits/a43eef893ad88d077f3b50fa43a34d06a949149c)" 

What does it mean? How other develpers will understand which comments were fixed? 
1. You should describe what exactly was implemented/fixed
2. All changes should not be in one commit - it should be multiple commits, where one commit - one logical unit of change